### PR TITLE
underline not working when hover question

### PR DESCRIPTION
**Pull request by jack-han-115**
Fixed issue #<Mantis issue number>:
New feature #<Mantis issue number>:
Dev:

### CODE REVIEW

File: assets/bootstrap_5/scss/customizations.scss [hunk 1]
```diff
@@ -1,38 +1,43 @@
 html {
-    font-size: $font-size-root;
+  font-size: $font-size-root;
 }
 
 a {
-    text-decoration: none;
+  text-decoration: none;
```
Comment (Shnoulle) [inline assets/bootstrap_5/scss/customizations.scss:6]: I don't understand such customisation : see https://github.com/twbs/bootstrap/blob/d9f8be677325d8060253dd25716764f37dcbd343/scss/_reboot.scss#L245

It's inside variable (and by default get underline)

BS5 is better for accessibility and usage than BS3, why broke it ?

File: assets/bootstrap_5/build/css/bootstrap_5.css [hunk 2]
```diff
@@ -10950,4 +10951,21 @@ a {
 
 .table > :not(:first-child) {
   border-top: 0;
+<<<<<<< HEAD
```
Comment (olleharstedt) [inline assets/bootstrap_5/build/css/bootstrap_5.css:10954]: Ah, this is not correct, I think.

**Review by olleharstedt — CHANGES_REQUESTED** [review]

File: assets/bootstrap_5/scss/customizations.scss (hunk 1 above)
Comment (olleharstedt) [reply assets/bootstrap_5/scss/customizations.scss:6]: We don't have accessibility for admin theme tho.

Comment (Shnoulle) [reply assets/bootstrap_5/scss/customizations.scss:6]: Not only a11Y , but 

1. BS5 have a variable for this : why we LS don't use variable (more easy to create new theme) why it's not `_variables.scss` here ? Or usage of variable ?
2. BS5 default is underline : https://github.com/twbs/bootstrap/blob/d9f8be677325d8060253dd25716764f37dcbd343/scss/_variables.scss#L450

Comment (olleharstedt) [reply assets/bootstrap_5/scss/customizations.scss:6]: Hm, @ptelu it sick, but he'll be back next week. I hope. :)

File: assets/bootstrap_5/build/css/bootstrap_5.css (hunk 2 above)
Comment (Shnoulle) [reply assets/bootstrap_5/build/css/bootstrap_5.css]: Are you sure ? 🤣

Comment (jack-han-115) [reply assets/bootstrap_5/build/css/bootstrap_5.css]: i will need to create new PR again for this issue.

**Pull request closed without merging**